cmd/flags.go

```diff
@@ -99,6 +102,38 @@ func getProfileList(cmd *cobra.Command) (profileList []string, err error) {
 	return profileList, nil
 }
 
+func getMaxConcurrency(cmd *cobra.Command) (maxConcurrency string, err error) {
+	defaultMaxConcurrency := "50"
```

Comment:
I think we can drop the usage of defaultMaxConcurrency in this function; if we can't fetch the flag value via `GetFlagString()`, we throw `log.Fatal()` from the caller anyway. Otherwise, the flag itself already provides the default value.